examples/online-store/build.gradle

```diff
@@ -5,7 +5,7 @@ buildscript {
         mavenLocal()
     }
     dependencies {
-        classpath("com.marklogic:ml-data-hub:${mlDHFVersion}")
+        classpath("com.marklogic:ml-data-hub:5.0-SNAPSHOT")

```

Comment:
I wasn't sure what to put here, as in order for the custom hook to work, 5.0-SNAPSHOT needs to be published locally so that the fix to hub-utils.sjs is available.